src/lib/components/html-report-page.jsx

```diff
@@ -20,37 +20,55 @@ export type FlowUncoveredLocsProps = {
   uncovered_locs: Array<FlowUncoveredLoc>
 };
 
-/* eslint-disable react/no-unused-prop-types */
-export type FlowCoverageReportProps = {
-  reportType: 'summary' | 'sourcefile',
+export type FlowReportMetaProps = {
+  assets: {
+    css?: Array<string>,
+    js?: Array<string>
+  },
   coverageGeneratedAt: string,
+  htmlTemplateOptions: {
+    autoHeightSource?: boolean,
+  },
+};
+
+export type FlowCoverageSummaryReportProps = {
   assets: {
     css?: Array<string>,
     js?: Array<string>
   },
+  coverageGeneratedAt: string,
+  htmlTemplateOptions: {
+    autoHeightSource?: boolean,
+  },
```

Comment:
@rpl I went with Meta because HTMLReportHead and HTMLReportFooter both take only these props. It's not like an abstract shared base class or something.

I think of inheritance when I use&see 'Base*', which isn't what we have here IMO. Rather, the Page takes the union of all the props that are required by the components, so we might need a name for the group of `coverageData, fileName, fileContent, summaryRelLink` fields that is more specific to the < HTMLReportBodySourceFile /> component.

I'm happy to refine names any time though, as we narrow the definitions for things, but I disagree with it this time.